includes/data-stores/class-wc-admin-reports-customers-data-store.php

```diff
@@ -474,32 +499,7 @@ public function get_guest_by_email( $email ) {
 		); // WPCS: unprepared SQL ok.
 
 		if ( $guest_row ) {
-			return $this->cast_numbers( $guest_row );
-		}
-
-		return false;
-	}
-
-	/**
-	 * Retrieve a registered customer row by user_id.
-	 *
-	 * @param string|int $user_id User ID.
-	 * @return false|array Customer array if found, boolean false if not.
-	 */
-	public function get_customer_by_user_id( $user_id ) {
-		global $wpdb;
-
-		$table_name = $wpdb->prefix . self::TABLE_NAME;
-		$customer   = $wpdb->get_row(
-			$wpdb->prepare(
-				"SELECT * FROM {$table_name} WHERE user_id = %d LIMIT 1",
-				$user_id
-			),
-			ARRAY_A
-		); // WPCS: unprepared SQL ok.
-
-		if ( $customer ) {
-			return $this->cast_numbers( $customer );
+			return $guest_row['customer_id'];
```

Comment:
We should only query for `customer_id` if that's all we're returning. Also this needs to be casted to an int.

Reply:
Nice!  Thanks for catching that.  Updated in https://github.com/woocommerce/wc-admin/pull/1619/commits/75a315fb947a05ea5d1c360cd35ade1ad9c2a868